### ISSUE TITLE
Node-red error sysmodule not load

### ISSUE THREAD

**Issue by Joe-ab1do** [redplc/node-red-contrib-redplc-rpi-mcp23008#1]
Hi,
I tried using your code to operate a mcp23008 chip. I installed all the required redPlc nodes with the node-red palette manager and tried to recreate the example you give as a start. However, when deploying the flow I get the error "sysmodule not load":
![node-red flow](https://user-images.githubusercontent.com/90150294/132137929-a378fdf8-24ce-4a29-ab94-aea9e057dd9c.jpg)

Node-red also shows no global context data. I looked through the mcp23008.js code and noticed this line:
const syslib = require('./lib/syslib.js');

A little further down (line47) the code tests if the syslib module was loaded and if not the error "sysmodule not load" is thrown, which is what I see. So it would seem syslib.js is missing or cannot be found. Digging further, I noticed that the syslib.js file is present as expected in my
~/.node-red/node_modules/node-red-contrib-redplc-rpi-mcp23008/nodes/lib directory. So now I am at a loss and don't know what to do to solve this issue. If there is anything you can do to point me in the right direction that would be great.

FYI, I am running node-red on a Raspberry Pi 4, 64bit operating system, node-red version 2.0.5, nodejs version 14.17.6, npm version 6.14.15

BTW, I have the same issue with the rpi-gpio node: sysmodule not load error is thrown, yet its syslib.js is present where expected.

Thanks,
Joe

**Comment by redplc** [comment]
Hi Joe,
for performance reason the node use "compiled binaries" with extension **.node**.
This binaries are compiled with 32bit compiler and are not suitable for RPI 64bit operating system.
Please use RPI 32bit operating system.
Sorry for malfunction of node.

**Comment by Joe-ab1do** [comment]
OK, I was about to do that anyway, because I ran into pretty much the same issue with another node. Luckily I only installed the 64bit operating system when I set up the Pi about a week ago, so not too much other software installed.

Now that a 64 bit operating system exists for the Pi (4), it might be an idea to add a comment to the documentation that the node will only work for 32 bit operating systems. Just a thought.

Thanks for the quick response!
